tests/network/test_reaction_network.py

```diff
@@ -561,7 +553,7 @@ def test_final_PR_check(self):
         # assert
         self.assertTrue(output.__contains__("No path found from any start to PR 30"))
         self.assertTrue(
-            output.__contains__("WARNING: Matching prereq and byproduct found! 46")
+            output.__contains__("WARNING: Matching prereq and byproduct found! 542")
```

Comment:
Let's change this warning to a "NOTE", or something like that. It's no longer a sign of something bad happening, but it's reasonable to think that the user might want to be aware of it. And I'm assuming that the matching PR/BP changed from 46 to 542 thanks to paths reordering since we aren't performing cancelation?